lib/mercury/cps.rb

```diff
@@ -80,9 +80,7 @@ def self.concurrently(*cpss)
           cps.run(*in_args) do |*out_args|
             returned_args[i] = out_args
             pending_completions.delete(cps)
-            if pending_completions.none?
-              k.call(returned_args)
-            end
+            k.call(returned_args) if pending_completions.none?
```

Comment:
There are already several levels of indentation here, and I think the flow is easier to see at a glance with extra indentation from the multi-line 'if', compared to having the 'if' appended.